tests/classifications/test_classification_signals.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from django.db.models import signals
+
+from adhocracy4.comments.models import Comment
+from apps.classifications.signals import get_ai_classification
+
+
+@pytest.mark.django_db
+def test_ai_classification_receives_post_save_comment_signal(idea,
+                                                             comment_factory,
+                                                             caplog):
+    assert(get_ai_classification in signals.post_save._live_receivers(Comment))
+    comment_factory(content_object=idea,
+                    comment='lala')
+    assert('Error connecting to %s' in str(caplog.records[-1]))
```

Comment:
So this is now just testing if the signal is received and that the ai call fails.. not optimal, but didnt find any other way to test it..

Reply:
This might give a hint on how to do it: https://realpython.com/testing-third-party-apis-with-mocks/
But having the tests you wrote is already awesome, so I will merge!